README.md

```diff
@@ -85,21 +85,25 @@ Initialize the hub by deploying the hub side resources to manage clusters.
 `clusteradm init [--use-bootstrap-token]`
 
 it returns the command line to launch on the spoke to join the hub.
+> NOTE: Do not run init command against a [multicluster-controlplane](https://github.com/open-cluster-management-io/multicluster-controlplane) instance. It is already an initialized hub on start. Instead, use `clusteradm get token --use-bootstrap-token` to get the join command.
 
 ### get token
 
 Get the latest token to import a new managed cluster.
 
 `clusteradm get token --context ${CTX_HUB_CLUSTER}`
-
 ### join
 
 Install the agent on the spoke.
 
-`clusteradm join --hub-token <token> --hub-apiserver <hub_apiserver_url> --cluster-name c1`
+`clusteradm join --hub-token <token> --hub-apiserver <hub_apiserver_url> --cluster-name c1 [--ca-file <path-to-ca-file>] [--force-internal-endpoint-lookup]`
 
 it returns the command line to launch on the hub the accept the spoke onboarding.
 
+> NOTE: While clusteradm could not fetch any CA data from given `--hub-apiserver`, then `--ca-file` flag should be set to provide a valid CA.
```

Comment:
we probably add more description here. The ca data is fetch from xx secret kube-public namespace it is not set.

Reply:
done